docs/manuals/admin/telemetry.md

```diff
@@ -0,0 +1,199 @@
+# Telemetry
+
```

Comment:
it seems not yet linked with the doc tree for readthedocs

Reply:
thx, done, see 9dfe08c.